src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java

```diff
@@ -44,4 +48,14 @@ public NewsDTO getById(Long id){
         return this.mapper.newsEntity2DTO(entityResult.get());
     }
 
+    @Transactional
+    @Override
+    public void deleteById(Long id) {
+        Optional<News> entity = this.newsRepository.findById(id);
+        if(entity.isEmpty()){
+            throw new NotFoundException("Id not found");
```

Comment:
En lugar de Id not found pondria News not found, o News with id provided not found, para que sea mas descriptivo.

Reply:
ya cambio ese mensaje, y conrepecto a lo anterior no se setiaron xq cree una news a mano en la base de datos se me paso por alto ese detalle ahora lo hago desde el endpont correpondiete

Reply:
![OT207-48](https://user-images.githubusercontent.com/84826653/172397322-edb45dae-47e6-48a4-a4df-5b163c23d354.jpg)